### ISSUE TITLE
--version, --help, usage

### ISSUE THREAD

**Issue by bukzor** [Yelp/dumb-init#34]
We should support these minimal expected cli behaviors:

```
$ dumb-init
usage: dumb-init <your program here>

$ dumb-init --version
0.5.0

$ dumb-init --help
usage: dumb-init <your program here>

`dumb-init` can serve as PID1 in a minimal system. It any recieved signals to its child.
This is particularly useful in the context of docker; it makes ctrl-c and friends Just Work. 
```

For bonus credit, make `-v` set debug to 1.

**Comment by chriskuehl** [comment]
Another benefit is so we can use help2man to generate a simple man page.

**Comment by bukzor** [comment]
--version being the most important one

**Comment by bukzor** [comment]
fixed in #37
